invertedai/logs/logger.py

```diff
@@ -499,26 +499,29 @@ def __init__(
         all_agent_states_unsorted = []
         all_agent_properties_unsorted = {}
         present_indexes_unsorted = []
+        agent_id_list = {}
+        curr_agent_index = 0
```

Comment:
Is this only for counting the sequence number for each agent_id ? Maybe `agent_id_sequence_num` is a better name ?